<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>673</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>400</height>
        </size>
       </property>
       <property name="title">
        <string>Setup</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="gpuGroupBox">
          <property name="toolTip">
           <string>Set the real GPU location into the machine</string>
          </property>
          <property name="title">
           <string>GPU location</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="gpuLocationLayout">
             <item>
              <widget class="QRadioButton" name="discreteRadioBtn">
               <property name="toolTip">
                <string>Click this if there is an AGP, PCI or PCI-Express graphics card</string>
               </property>
               <property name="text">
                <string>Discrete</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="intMoboRadioBtn">
               <property name="toolTip">
                <string>Click this if the GPU is located into the motherboard</string>
               </property>
               <property name="text">
                <string>Integrated in Motherboard</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="intCpuRadioBtn">
               <property name="toolTip">
                <string>Click this if the GPU is located into the CPU</string>
               </property>
               <property name="text">
                <string>Integrated in CPU</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="componentsGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Select the components that you want to send to TARALLO</string>
          </property>
          <property name="title">
           <string>Selectors</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QScrollArea" name="selectorsScrollArea">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="selectorsWidget">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>-33</y>
                <width>221</width>
                <height>298</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QCheckBox" name="checkBox_2">
                 <property name="text">
                  <string>Case</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox">
                 <property name="text">
                  <string>Motherboard</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_3">
                 <property name="text">
                  <string>CPU</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_7">
                 <property name="text">
                  <string>GPU</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_4">
                 <property name="text">
                  <string>RAM</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_6">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Hard disk</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_5">
                 <property name="text">
                  <string>ODD</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_11">
                 <property name="text">
                  <string>PSU</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_8">
                 <property name="text">
                  <string>Monitor</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_10">
                 <property name="text">
                  <string>Keyboard</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_9">
                 <property name="text">
                  <string>Mouse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="toolTip">
           <string>Set the owner of the computer</string>
          </property>
          <property name="title">
           <string>Owner</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLineEdit" name="ownerLineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="resetBtn">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="generateBtn">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Results</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>478</width>
             <height>540</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QToolBox" name="toolBox">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="default_page">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>460</width>
                 <height>489</height>
                </rect>
               </property>
               <attribute name="label">
                <string>...</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="saveJsonBtn">
            <property name="text">
             <string>Save JSON</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uploadBtn">
            <property name="text">
             <string>Upload</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenJson"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionTheme"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutUs"/>
    <addaction name="actionSourceCode"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuInfo"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="text">
    <string>Theme</string>
   </property>
  </action>
  <action name="actionAboutUs">
   <property name="text">
    <string>About WEEE Open</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionOpenJson">
   <property name="text">
    <string>View JSON</string>
   </property>
  </action>
  <action name="actionSourceCode">
   <property name="text">
    <string>Source code</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
